src/main/java/com/bookstore/service/SecurityUserDetailsService.java

```diff
@@ -0,0 +1,22 @@
+package com.bookstore.service;
+
+import com.bookstore.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Primary;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@Primary
```

Comment:
You have several implementation of UserDetailsService and you need to specify which of should be injected?